docs/development/getting_started.rst

```diff
@@ -128,6 +137,7 @@ different version, the path to ``virtualenvwrapper.sh`` will differ. Running
     source /usr/local/bin/virtualenvwrapper.sh
     mkvirtualenv -p python2 securedrop
     pip install -r securedrop/requirements/develop-requirements.txt
+    pip install -r testinfra/requirements.txt
```

Comment:
While testing this PR branch, I intuitively reran the `develop-requirements` files and confirmed that Molecule was _not_ added, even though it's a requirement for development now. We should fix the requirements story quite soon. Willing to let it pass here and in #2218, and continue tracking in #1918 & #2175.